java/hello-world/README.md

```diff
@@ -3,20 +3,132 @@
+
+These samples use the [Apache Maven][maven] build system. Before getting
+started, be sure to [download][maven-download] and [install][maven-install] it.
+When you use Maven as described here, it will automatically download the needed
+client libraries.
+
+[maven]: https://maven.apache.org
+[maven-download]: https://maven.apache.org/download.cgi
+[maven-install]: https://maven.apache.org/install.html
+
+### Creating a Project in the Google Cloud Platform Console
+
+If you haven't already created a project, create one now. Projects enable you to
+manage all Google Cloud Platform resources for your app, including deployment,
+access control, billing, and services.
+
+1. Open the [Cloud Platform Console][cloud-console].
+1. In the drop-down menu at the top, select **Create a project**.
+1. Give your project a name.
+1. Make a note of the project ID, which might be different from the project
+   name. The project ID is used in commands and in configurations.
+
+[cloud-console]: https://console.cloud.google.com/
+
+### Enabling billing for your project.
+
+If you haven't already enabled billing for your project, [enable
+billing][enable-billing] now.  Enabling billing allows the application to
+consume billable resources such as running instances and storing data.
+
+[enable-billing]: https://console.cloud.google.com/project/_/settings
+
+### Install the Google Cloud SDK.
+
+If you haven't already installed the Google Cloud SDK, [install the Google
+Cloud SDK][cloud-sdk] now. The SDK contains tools and libraries that enable you
+to create and manage resources on Google Cloud Platform.
+
+[cloud-sdk]: https://cloud.google.com/sdk/
+
+### Setting Google Application Default Credentials
+
+Set your [Google Application Default
+Credentials][application-default-credentials] by [initializing the Google Cloud
+SDK][cloud-sdk-init] with the command:
+
+		gcloud init
+
+Alternatively, create a service account key and set the
+`GOOGLE_APPLICATION_CREDENTIALS` environment variable to the service account
+key file path.
+
+[cloud-sdk-init]: https://cloud.google.com/sdk/docs/initializing
+[application-default-credentials]: https://developers.google.com/identity/protocols/application-default-credentials
+
+
+## Provisioning a cluster
+
+Follow the instructions in the [user
+documentation](https://cloud.google.com/bigtable/docs/creating-cluster) to
+create a Google Cloud Platform project and Cloud Bigtable cluster if necessary.
+You'll need to reference your project id, zone and cluster id to run the
+application.
+
+
+## Running the application
+
+Set the following environment variables or replace them with the appropriate
+values in the `mvn` commands. Set:
+
++   `GCLOUD_PROJECT` to the project ID,
++   `BIGTABLE_CLUSTER` to the Bigtable cluster ID,
++   `BIGTABLE_ZONE` to the Bigtable compute zone (example: us-central1-b).
+
+Build and run the sample using Maven.
+
+    mvn package
+    mvn exec:java -Dbigtable.projectID=${GCLOUD_PROJECT} \
```

Comment:
Working on this now, will be -Dbigtable.projectID=... & -Dbigtable.instanceID=  removing clusterID & zone